internal/entimport/import_test.go

```diff
@@ -3930,6 +3917,14 @@ type Inspector struct {
 	mock.Mock
 }
 
+func (_m *Inspector) InspectTable(ctx context.Context, name string, opts *schema.InspectTableOptions) (*schema.Table, error) {
+	return nil, nil
+}
+
+func (_m *Inspector) InspectRealm(ctx context.Context, opts *schema.InspectRealmOption) (*schema.Realm, error) {
+	return nil, nil
```

Comment:
Same here.

Reply:
it's a mock, I must implmenet the interface and there is no usage for this func.